### ISSUE TITLE
bootlogo package is not included on extras on fulan targets (regression)

### ISSUE THREAD

**Issue by lprot** [PLi-metas/pli-extras#49]

**Comment by lprot** [comment]
There's no signs of bootlogo in opkg lists, no showiframe in /usr/bin etc

**Comment by kueken** [comment]
Yes, that's not a big deal.
Incidentally, a boot logo is displayed, just the bare black bootlogo.
It did not seem so urgent to me because anyone can locally replace the boot logo.
But I put a bootlogo there.

**Comment by lprot** [comment]
Heh. That way image is becoming a lego. Too many things you need to fix after compilation :(

**Comment by kueken** [comment]
I do not know what to do now?
That there is no standard boot logo you have already written. So I closed the double post again.
In addition, many things have been addressed and are already running. But you are never satisfied.
When solving some things you always have to be careful not to destroy existing structures.
If you have a simple solution and are familiar with it, then you are welcome to pry it into git.
I have no problem with that.
Because of bootlogo black or with 4 words on a black background making such a noise is already ungrateful.
